thorlcr/thorutil/thmem.hpp

```diff
@@ -88,7 +88,7 @@ class OwnedConstThorRow
     inline OwnedConstThorRow(const void * _ptr)             { TESTROW(_ptr); ptr = _ptr; }
     inline OwnedConstThorRow(const OwnedConstThorRow & other)   { ptr = other.getLink(); }
 
-    inline ~OwnedConstThorRow()                             { ReleaseThorRow(ptr); }
+    inline virtual ~OwnedConstThorRow()                     { ReleaseThorRow(ptr); }
```

Comment:
I agree - more correct would be for CThorRowHistory to *contain* a row, not inherit from it.  I'm not convinced it has a is-a relationship.